crates/router/src/types/transformers.rs

```diff
@@ -526,7 +526,7 @@ impl ForeignFrom<storage::Dispute> for api_models::disputes::DisputeResponsePaym
             challenge_required_by: dispute.challenge_required_by,
             created_at: dispute.dispute_created_at,
             updated_at: dispute.updated_at,
-            received_at: dispute.created_at.to_string(),
+            received_at: dispute.created_at,
```

Comment:
I feel this mapping would be a possible source of confusion and bugs (due to incorrect mapping in a different place) on a later date. 

Can we avoid it, and map fields with the same names instead? I understand this would be an API contract change/database schema change, but it induces an additional cognitive load on the developer working with this.

```rust
created_at: dispute.dispute_created_at,
received_at: dispute.created_at,
```